agenda/data/src/main/java/com/gkp/agenda/data/image/ImageCompressor.kt

```diff
@@ -0,0 +1,15 @@
+package com.gkp.agenda.data.image
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import java.io.ByteArrayOutputStream
+
+object ImageCompressor {
+
+    fun compressImage(imageBytes: ByteArray, quality: Int = 50): ByteArray {
```

Comment:
Could changing this to a `suspend` function make more sense? This function will block the main thread when the caller does not move the execution to a background thread. Can you think of a way to make this function more main-safe?